src/Spreadaccacadmey/SpreadAcademy.js

```diff
@@ -0,0 +1,78 @@
+import {
+  GradiendtTextWrapper,
+  IconContainar,
+  Description,
+  EmtyDiv,
+  LeftWrapper,
+  LeftSideIcon,
+  SpreadAcademyContainer,
+  SpreadAcademyWrapper,
+  SpreadAcademySection,
+  SpreadAcademyDescription,
+  RightWrapper,
+  RightWrapperAfter,
+  RightWrapperBefore,
+  RightCardImage,
+  SmallCard,
+  CardImage,
+  SpreadAcademyTitle,
+  CardText,
+} from "./SpreadAcademy.styled";
+
+export function SpreadAcademy() {
+  return (
+    <div>
+      <SpreadAcademySection>
+        <IconContainar>
+          <LeftSideIcon
+            src="https://spreadd.io/assets/home/left-dot-patch.svg"
+            alt="Dot icon"
+          />
+        </IconContainar>
+        <SpreadAcademyContainer>
+          <SpreadAcademyWrapper>
+            <LeftWrapper>
+              <GradiendtTextWrapper>Spreadd Academy</GradiendtTextWrapper>
+              <SpreadAcademyTitle>Create course and</SpreadAcademyTitle>
```

Comment:
make this all title in one line and adjust with design with fixed max-width @PRABgit01